data/strings/strings.txt

```diff
@@ -21073,3 +21073,172 @@
     hi = बाद में मैनुअल रूप से अपडेट करें
     sk = Manuálne aktualizovať neskôr
     sw = Sasisha mwenyewe baadaye
+
+  [placepage_delete_track_button]
```

Comment:
Could you please squash and rebase this PR on the latest master?

```
git fetch origin -a
git rebase origin/master
# Resolve conflicts:
git checkout origin/master -- iphone/Maps/LocalizedString
git rebase --continue
./tool/unix/generate_localization.sh
git rebase -i HEAD~<number of commits> # create one commit
# Fixup
```

We had a problem with localization a couple of days ago. This problem has been solved already, but all PRs now conflicts in phone/Maps/LocalizedStrings. I didn't have localization strings until I did step above :(

Reply:
The easier way to fix conflict may be an interactive rebase with `git rebase -i origin/master`, it allows to edit commit or remove it entirely. That is the reason why we separate strings.txt commits from generated localizations commits.

Reply:
I've squashed all my commits into one and rebased it onto master.

I also regenerated localizations from strings.txt and placed it to separate commit (not sure it should be done on developer's side, so it can be reverted).

P.S. I've manually added english texts for new keys for all languages I don't know (e.g. bulgarian, etc), but I'm not sure it is correct. Does generate_localizations add missing translations "as english" by default or should they be added manually?

Thanks